assets/static/js/index.js

```diff
@@ -0,0 +1,20 @@
+const urlParams = new URLSearchParams(window.location.search);
+
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Please use 2 spaces instead of 4. We don't have a javascript style guide yet (sorry).